hack/dev-rook-cluster.yaml

```diff
@@ -9,31 +11,50 @@ data:
     [global]
     osd_pool_default_size = 1
     mon_warn_on_pool_no_redundancy = false
+    bdev_flock_retry = 20
+    bluefs_buffered_io = false
 ---
 apiVersion: ceph.rook.io/v1
 kind: CephCluster
 metadata:
-  name: rook-ceph
+  name: my-cluster
   namespace: rook-ceph
 spec:
   dataDirHostPath: /var/lib/rook
   cephVersion:
-    image: ceph/ceph:v15
+    image: quay.io/ceph/ceph:v16.2.6
     allowUnsupported: true
   mon:
     count: 1
     allowMultiplePerNode: true
+  mgr:
+    count: 1
+    allowMultiplePerNode: true
   dashboard:
     enabled: true
   crashCollector:
     disable: true
   storage:
     useAllNodes: true
     useAllDevices: true
-  network:
-    provider: host
   healthCheck:
     daemonHealth:
       mon:
         interval: 45s
         timeout: 600s
+  disruptionManagement:
+    managePodBudgets: true
+  network:
+    provider: host
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: device-health-metrics
```

Comment:
Why is this new blockpool needed?

Reply:
No idea, copied from upstream rook and retained as is.